test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -1226,8 +1226,11 @@ func deleteNodePool(name string) {
 
 func getPoolNodes(f *framework.Framework, poolName string) []*v1.Node {
 	nodes := make([]*v1.Node, 0, 1)
-	nodeList, err := e2enode.GetReadyNodesIncludingTaintedOrDie(f.ClientSet)
-	framework.ExpectNoError(err)
+	nodeList, err := e2enode.GetReadyNodesIncludingTainted(f.ClientSet)
+	if err != nil {
+		e2elog.Logf("Unexpected error occurred: %v", err)
+	}
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	for _, node := range nodeList.Items {
```

Comment:
I got the error from prow `e2e-verify`:

`The above files need to use framework.ExpectNoError(err) instead of 
Expect(err).NotTo(HaveOccurred()) or gomega.Expect(err).NotTo(gomega.HaveOccurred())`
Thought that we can ignore this?

Reply:
The reason for this seems to have been [hack/verify-test-code.sh](https://github.com/kubernetes/kubernetes/blob/master/hack/verify-test-code.sh) which I didn't know it existed.
I don't know if there is a broader conversation about instances where we can ignore this as it seems rather arbitrary.

Under the premise that we keep `hack/verify-test-cdoe.sh` as is we can:
1. Create a wrapper in the framework for `gomega.Expect(err).NotTo(gomega.HaveOccurred())`, something like:
```go
func ExpectNoErrorToOccur(err error, explain ...interface{}) {
	ExpectNoErrorWithOffset(0, err, explain...)
}
```
2. Call `framework. ExpectNoErrorWithOffset(0, err)`

Reply:
I'd like to use the second way and leave a TODO in the comment. I was thinking open a tracking issue to track all the PRs will a TODO after we moving and renaming.

Reply:
+1 to that!